examples/ASP.NET Core 2/Visual Studio 2017/ASP.NET Core 2 - VS2017/Program.cs

```diff
@@ -11,7 +11,7 @@ public static class Program
     {
         public static void Main(string[] args)
         {
-            var logger = NLog.LogManager.LoadConfiguration("nlog.config").GetCurrentClassLogger();
+            var logger = NLog.Web.NLogBuilder.ConfigureNLog("nlog.config").GetCurrentClassLogger();
```

Comment:
mmm maybe this needs a minor version change, because of changed api usage.

(4.6 instead of 4.5.1)

Reply:
As the two bug-issues has shown, then 4.5.0 is not working that well out of the box (actually very broken). Think 4.5.1 would be fine. Unless you have a good idea for how to pre-register when AspNetCore?

Reply:
OK, thanks, `NLog.LogManager.LoadConfiguration` still works of course (in the same bugged way)